elb.tf

```diff
@@ -27,14 +27,13 @@ resource "aws_security_group_rule" "elb_egress" {
 resource "aws_security_group_rule" "elb_ingress" {
   count = length(var.elb_allowed_cidr_blocks) > 0 ? 1 : 0
 
-  cidr_blocks              = var.elb_allowed_cidr_blocks #tfsec:ignore:AWS007
-  description              = "Allow traffic from the allowed ranges"
-  from_port                = 443
-  protocol                 = "tcp"
-  security_group_id        = aws_security_group.elb.id
-  source_security_group_id = aws_security_group.this.id
```

Comment:
Why are you removing the source security group here?

Reply:
you can't mix `source_security_group_id` in the same `aws_security_group_rule` with `cidr_blocks`. This was the primary issue I was fixing in this PR. I honestly didn't realize `elb-tagging` had been merged already and reopened the branch applying this fix. Too many things going on at once :(